examples/simple_spgemm.cpp

```diff
@@ -24,14 +24,14 @@ int main(int argc, char** argv) {
 
   csr_view<float> c(nullptr, c_rowptr.data(), nullptr, {m, n}, 0);
 
-  auto info = multiply_inspect(a, b, c);
+  auto info = multiply_inspect(scaled(1.f, a), b, c);
```

Comment:
what happens if we put just "a" in instead of "scaled(1.f, a)" ?  Does it add the onefp value internally in the call to MKL ?

Reply:
For SpMV and SpMM, yes, the value `1` is passed in for `alpha` [in the call to MKL](https://github.com/SparseBLAS/spblas-reference/blob/onemkl-backend/include/spblas/vendor/mkl/multiply_impl.hpp#L54).

For SpGEMM, a scale is currently done after-the-fact, but [only if there's a `scale_view` present](https://github.com/SparseBLAS/spblas-reference/blob/onemkl-backend/include/spblas/vendor/mkl/multiply_impl.hpp#L199).